File: config/eventcount.cc

#define _USE_MATH_DEFINES

#include "TTree.h"
#include "TNtuple.h"
#include <fstream>
#include <sstream>
#include <sys/stat.h>
#include <errno.h>
#include <iostream>
#include <cstdlib>
#include "TChain.h"

using namespace std;


int main(){

	cout<<"start"<<endl;
	TChain ch("EventTree");
	ch.Add("/pnfs/iihe/cms/store/user/heilman/FOURTOP_NLO_13TeV_aMCatNLO_LHE_pythia8/crab_TOPTREE_742_FOURTOP_1M_v3/150605_093838/0000/*");
	cout<<"added files"<<endl;
	int nEntries = ch.GetEntries();
	cout<<nEntries<<endl;

	return 0;
}

File: config/eventcount.py
from ROOT import TChain
from glob import glob

path = '/pnfs/iihe/cms/store/user/fblekman/TopTree/CMSSW_76X_v3/TTTT_TuneCUETP8M1_13TeV-amcatnlo-pythia8/crab_atnlopythia8Fall15PU25nsData2015v176XmcRun2asymptoticv12ext1v176XmcRun2asymptoticFall15DR76v1crab35/160531_182635/0000/*.root'
files = glob(path)
root_files = []
for f in files:
	root_files.append('dcap://maite.iihe.ac.be' + f)
print root_files
chain = TChain('eventTree')
for rf in root_files:
	chain.Add(rf)
print 'added files'
nEntries = chain.GetEntries();
print nEntries

